packages/backend-common/src/logging/rootLogger.test.ts

```diff
@@ -29,4 +29,79 @@ describe('rootLogger', () => {
       expect.stringContaining('testing'),
     );
   });
+
+  describe('createRootLoger', () => {
+    beforeEach(() => {
+      delete process.env.LOG_STACKTRACE;
+    });
+
+    it('creates a new logger', () => {
+      const oldLogger = getRootLogger();
+      const newLogger = createRootLogger();
+
+      expect(oldLogger).not.toBe(newLogger);
+    });
+
+    it('replaces the existing root logger', () => {
+      const oldLogger = getRootLogger();
+      createRootLogger();
+      const newLogger = getRootLogger();
+      expect(oldLogger).not.toBe(newLogger);
+    });
+
+    it('can append additional default metadata', () => {
+      const format = winston.format.json();
+      const logger = createRootLogger({
+        format,
+        defaultMeta: {
+          appName: 'backstage',
+          appEnv: 'prod',
+          containerId: 'abc',
+        },
+      });
+      jest.spyOn(format, 'transform');
+
+      logger.info('testing');
+
+      expect(format.transform).toHaveBeenCalledWith(
+        expect.objectContaining({
+          message: 'testing',
+          appName: 'backstage',
+          appEnv: 'prod',
+          containerId: 'abc',
+        }),
+        {},
+      );
+    });
+
+    it('log entry should not include stack default', () => {
+      const format = winston.format.json();
+      const logger = createRootLogger({ format });
+      jest.spyOn(format, 'transform');
+
+      logger.error('testing', Error('error'));
+
+      expect(format.transform).not.toHaveBeenCalledWith(
```

Comment:
sure this should be not? the expectation looks a bit odd

Reply:
Yup